docs/install.md

```diff
@@ -62,7 +62,7 @@ Create a namespace for the deployed components:
 kubectl create namespace mlrun
 ```
 
-Add the `v3io-stable` helm chart repo:
+Add the `v3io-stable` helm chart repo (helm installation [guidelines](https://helm.sh/docs/intro/install/) and [sources](https://github.com/helm/helm/releases/)):
```

Comment:
@george0st note that this doc has the "Prerequisites" section, mentioning that you need to install the Helm CLI with a link
I feel like it is better to have all the prerequisites with the links arranged in one place, WDYT ?
Maybe just add the sources link there (instead of this line change) ?

Reply:
Done